### ISSUE TITLE
Loading a trained PFN model

### ISSUE THREAD

**Issue by WilliamBidle** [pkomiske/EnergyFlow#10]
Hi,
I am having an issue reloading a trained PFN model to use for later predictions. I have the whole model being saved at the end (not while training), and have tried loading it using load_model(), resulting in: "AttributeError: 'NoneType' object has no attribute 'get'". I am not sure if there is a specific way to load your saved models but if you have any help it would be greatly appreciated.

**Comment by ericmetodiev** [comment]
Hi William,

I tried to reproduce this issue using our PFN example after training for 1 epoch, but I don't run into any issue doing `model.save('test')` and then `load_model('test')`.

Can you give us a minimal example that reproduces your issue? Information about your setup (python version, operating system, Keras/tensorflow/energyflow versions, etc.) would be helpful as well.

Best,
Eric

**Comment by WilliamBidle** [comment]
Hi Eric,

From what I see in the archbase.py file, passing the hyperparameters 'filepath = model.h5' and 'save_while_training = false' into the PFN class should save the model to the filepath I specified. Would it just be easier to save the model directly like you had mentioned? It just seemed as though I had to pass a few hyperparameters to save the model. I used the following:
pfn = PFN(input_dim=X.shape[-1], Phi_sizes=Phi_sizes, F_sizes=F_sizes, save_while_training = False, filepath='model.h5')

I am using:
Python - 3.7.3
Operating System - MacOS
Keras - 2.2.4
Tensorflow - 1.13.1
Energyflow - 0.12.1

**Comment by ericmetodiev** [comment]
Hi William,

Thank you for the details! I reproduced the problem: There does appear to be a subtle issue with easily saving and loading models as implemented in Energyflow `0.12.1`. We made changes that resolved the issue, which were released in `0.12.3`. Running on the latest version of Energyflow (`0.13.0`), your syntax works for me.

Let me know if updating the version of Energyflow resolves your issue.

Best,
Eric
